build/phpstan.neon

```
@@ -115,6 +115,11 @@ parameters:
 			count: 1
 			path: ../src/Diagnose/PHPStanDiagnoseExtension.php
 		- '#^Short ternary operator is not allowed#'
+		-
+			identifier: property.unusedType
+			message: '#\(bool\) is never assigned (false|true) so it can be removed from the property type#'
+			path: ../tests/PHPStan/*RuleTest.php
+
```

we cannot use true/false property types in tests, as the simple-downgrader cannot downgrade them.
therefore ignore the errors